### PR TITLE
Saml auth

### PR DESCRIPTION
**Pull request by kvb2univpitt**

### CODE REVIEW

**Review by jklann — CHANGES_REQUESTED** [review]
Very hard to tell what if anything changed in the second commit with all the formatting changes. Maybe you could give line numbers? Or is it easier to cross-reference with your repository?

**Comment by kvb2univpitt** [comment]
@jklann @mikemendis I reverted all the changes in the repo.  I then apply all of my changes without reformatting the code and pushed to the repo.  Now, you can clearly see where all the changes are made.

File: edu.harvard.i2b2.pm/src/edu/harvard/i2b2/pm/delegate/ServicesHandler.java [hunk 1]
```diff
@@ -806,7 +821,7 @@ private String runGetAllDatasource(PMDbDao pmDb, String project, String caller)
                                                 } else if (f3.getName().equals("jta")) {
                                                     datasource.setJta(Boolean.valueOf(f3.get(obj2).toString()));
                                                 }
-                                                log.debug(f3.getName());
+                                                System.out.println(f3.getName());
```
[the remark this answers is not in the document]
Comment (kvb2univpitt) [reply edu.harvard.i2b2.pm/src/edu/harvard/i2b2/pm/delegate/ServicesHandler.java]: I reverted all the changes.

[the remark this answers is not in the document]
Comment (jklann) [reply edu.harvard.i2b2.pm/src/edu/harvard/i2b2/pm/delegate/ServicesHandler.java]: I still think you mean log.debug instead of println?

**Review by jklann — COMMENTED** [review]
Code changes look good (except the one println change). Do we need the final commit (ecf99eb324e72dacb881c37d61ccf7bcffbffd2b) that does reformatting?

**Comment by kvb2univpitt** [comment]
> Code changes look good (except the one println change). Do we need the final commit ([ecf99eb](https://github.com/i2b2/i2b2-core-server/commit/ecf99eb324e72dacb881c37d61ccf7bcffbffd2b)) that does reformatting?

That println is no long there.

**Pull request merged**